### DISCUSSION TITLE
Improve Device Remove Response

### DISCUSSION THREAD

**Discussion by steakpowered** [dechamps/LGTVDeviceListener#6] [Ideas]
I've been using this for a while and it's been working great.  I did some experimenting and found that if I was able to improve the remove time by about 1 second.

Steps:

1. Start in discovery mode.
2. Toggle the kvm and noting down the first instance of device added and device removed.
3. Create two separate instances with the devices from step 2.  One for add-input and one for remove-input

Although this can be achieved by manually running the application, but I'd like to run it as a service.

Enhancement suggestions:
1. Provide the capability to take a device name for removing.
or
2. Allow for a name parameter when creating the service

**Comment by dechamps** [comment]
That's interesting, filed #7 to track the feature request.

For the time being, as a workaround you should be able to run two services by setting up one service and then renaming it by manipulating the registry - specifically, by renaming the `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\LGTVDeviceListener` key - before setting up the other. I haven't tried it, but there's no reason it wouldn't work.
